pkg/security/probe/process_killer.go

```diff
@@ -700,7 +712,7 @@ const (
 )
 
 type ruleDisarmer struct {
-	sync.Mutex
+	m               sync.Mutex
```

Comment:
was embedding the sync.Mutex causing any issue ?

Reply:
No, adding a field name simply makes it easier for the IDE to find where the mutex is used